package.json

```diff
@@ -16,6 +16,7 @@
     "build:ts": "tsc",
     "build:ts:defs": "tsc --declaration --emitDeclarationOnly",
     "build:watch": "tsc --watch",
+    "repl": "ts-node -D 7017 -D 2393",
```

Comment:
- `2393` is the important one (ignore duplicate function)
- sure would be nice to have comments in json eh?